pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/state/StateContext.java

```diff
@@ -18,15 +18,61 @@
  */
 package org.apache.pulsar.functions.instance.state;
 
+import java.nio.ByteBuffer;
 import java.util.concurrent.CompletableFuture;
 
 /**
  * A state context per function.
  */
 public interface StateContext {
 
+    /**
+     * Increment the given <i>key</i> by the given <i>amount</i>.
+     *
+     * @param key key to increment
+     * @param amount the amount incremented
+     */
     void incr(String key, long amount);
```

Comment:
Should we instead do this as incrCount and getCount? Currently we have incr and getCount which dont seem to be related with each other.

Reply:
sure.